### PR TITLE
WIP: feature/cilium

### PR DESCRIPTION
**Pull request by mlauke**

### CODE REVIEW

**Comment by dploeger** [comment]
@mlauke Sorry, we lost track of this one. If this is still required, could you please rebase on develop? Thanks.

File: feature/git/install.sh
Comment (dploeger) [inline feature/git/install.sh:1]: I'd rather use the package feature to install git than a separate feature. Or is there a specific use for the git feature?

File: feature/cilium/feature.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+icon: ""
```
Comment (dploeger) [inline feature/cilium/feature.yaml:1]: Any idea for an icon? Maybe 𐄳? 😄

**Review by dploeger — CHANGES_REQUESTED** [review]
Great! Just two small remarks and please rebase and fixup the commits to just one. Thanks!

**Comment by dploeger** [comment]
Closing this for the time being

File: feature/cilium/feature.yaml (hunk 1 above)
Comment (ThomasMinor) [reply feature/cilium/feature.yaml:1]: grafik
 https://cilium.io/favicon-32x32.png

**Pull request closed without merging**